deployment/stacks/parent.ts

```diff
@@ -68,63 +68,72 @@ export class ParentStack extends Stack {
       brokerString,
     });
 
-    const {
-      applicationEndpointUrl,
-      cloudfrontDistributionId,
-      distribution,
-      s3BucketName,
-      uiBucket,
-    } = createUiComponents({
-      deploymentConfigParameters,
-      ...commonProps,
-    });
-
-    const {
-      userPoolDomainName,
-      identityPoolId,
-      userPoolId,
-      userPoolClientId,
-    } = createUiAuthComponents({
-      ...commonProps,
-      oktaMetadataUrl,
-      applicationEndpointUrl,
-      restApiId,
-      bootstrapUsersPasswordArn,
-      customResourceRole,
-    });
+    if (process.env.CDK_DEFAULT_ACCOUNT !== "000000000000") {
+      const {
+        applicationEndpointUrl,
+        cloudfrontDistributionId,
+        distribution,
+        s3BucketName,
+        uiBucket,
+      } = createUiComponents({
+        deploymentConfigParameters,
+        ...commonProps,
+      });
 
-    deployFrontend({
-      ...commonProps,
-      uiBucket,
-      distribution,
-      apiGatewayRestApiUrl,
-      applicationEndpointUrl,
-      identityPoolId,
-      userPoolId,
-      userPoolClientId,
-      userPoolClientDomain: `${userPoolDomainName}.auth.${this.region}.amazoncognito.com`,
-      customResourceRole,
-    });
+      const {
+        userPoolDomainName,
+        identityPoolId,
+        userPoolId,
+        userPoolClientId,
+      } = createUiAuthComponents({
+        ...commonProps,
+        oktaMetadataUrl,
+        applicationEndpointUrl,
+        restApiId,
+        bootstrapUsersPasswordArn,
+        customResourceRole,
+      });
 
-    new ssm.StringParameter(this, "DeploymentOutput", {
-      parameterName: `/${project}/${stage}/deployment-output`,
-      stringValue: JSON.stringify({
+      deployFrontend({
+        ...commonProps,
+        uiBucket,
+        distribution,
         apiGatewayRestApiUrl,
         applicationEndpointUrl,
-        s3BucketName,
-        cloudfrontDistributionId,
         identityPoolId,
         userPoolId,
         userPoolClientId,
         userPoolClientDomain: `${userPoolDomainName}.auth.${this.region}.amazoncognito.com`,
-      }),
-      description: `Deployment output for the ${stage} environment.`,
-    });
+        customResourceRole,
+      });
 
-    new CfnOutput(this, "CloudFrontUrl", {
-      value: applicationEndpointUrl,
-    });
-    
+      new ssm.StringParameter(this, "DeploymentOutput", {
+        parameterName: `/${project}/${stage}/deployment-output`,
+        stringValue: JSON.stringify({
+          apiGatewayRestApiUrl,
+          applicationEndpointUrl,
+          s3BucketName,
+          cloudfrontDistributionId,
+          identityPoolId,
+          userPoolId,
+          userPoolClientId,
+          userPoolClientDomain: `${userPoolDomainName}.auth.${this.region}.amazoncognito.com`,
+        }),
+        description: `Deployment output for the ${stage} environment.`,
+      });
```

Comment:
Since this is inside a localstack context, do we need to adjust these values as you did here: https://github.com/Enterprise-CMCS/macpro-mdct-seds/pull/15058/files#diff-06fcc7a4ca1aadf07dacee3ad767f6ee8ce9d6acc517266db5932125636a45d1R33